cloud_governance/zombie_cluster/delete_ec2_resources.py

```diff
@@ -41,6 +42,7 @@ def __init__(self, client: BaseClient, elb_client: BaseClient, elbv2_client: Bas
         self.elb_client = elb_client
         self.elbv2_client = elbv2_client
         self.get_detail_list = Utils().get_details_resource_list
+        self.SLEEP_TIME = 30
```

Comment:
Constant should not placed in init.
You should declare it on the class, see https://stackoverflow.com/questions/10672419/class-constants-in-python